tests/typecheck/contrib/admin/test_decorators.yml

```diff
@@ -73,6 +73,11 @@
     @admin.action(description="Some text here", permissions=["test"])
     def freestanding_action_fancy(modeladmin: "MyModelAdmin", request: HttpRequest, queryset: QuerySet[MyModel]) -> None: ...
 
+    @admin.action
+    def freestanding_action_http_response(modeladmin: "MyModelAdmin", request: HttpRequest, queryset: QuerySet[MyModel]) -> HttpResponse: ...
+
+    @admin.action
+    def freestanding_action_file_response(modeladmin: "MyModelAdmin", request: HttpRequest, queryset: QuerySet[MyModel]) -> FileResponse: ...
```

Comment:
Also add the new actions to the `actions = [...]` list below to ensure they are accepted there.